react/README.md

```diff
@@ -0,0 +1,218 @@
+    ```javascript
+    // bad
+    const Footer = require('./Footer/Footer.jsx')
+
+    // bad
+    const Footer = require('./Footer/index.jsx')
+
+    // good
+    const Footer = require('./Footer')
+    ```
+
+
+## Declaration
+  - Do not use displayName for naming components, instead name the component by reference.
+
+    ```javascript
+    // bad
+    export default React.createClass({
+      displayName: 'ReservationCard',
+      // stuff goes here
+    });
+
+    // good
+    const ReservationCard = React.createClass({
+      // stuff goes here
+    });
+    export default ReservationCard;
+    ```
+
+## Alignment
+  - Follow these alignment styles for JSX syntax
+
+    ```javascript
+    // bad
+    <Foo bar="bar"
+         baz="baz" />
+
+    // good
+    <Foo
+      bar="bar"
+      baz="baz"
+    />
+
+    // if props fit in one line then keep it on the same line
+    <Foo bar="bar" />
+
+    // children get indented normally
+    <Foo
+      bar="bar"
+      baz="baz"
+    >
+      <Spazz />
+    </Foo>
+    ```
+
+## Quotes
+  - Always use double quotes (`"`) for JSX attributes, but single quotes for all other JavaScript.
+    ```javascript
+    // bad
+    <Foo bar='bar' />
+
+    // good
+    <Foo bar="bar" />
+
+    // bad
+    <Foo style={{ left: "20px" }} />
+
+    // good
+    <Foo style={{ left: '20px' }} />
+    ```
+
+## Spacing
+  - Always include a single space in your self-closing tag.
+    ```javascript
+    // bad
+    <Foo/>
+
+    // very bad
+    <Foo                 />
+
+    // bad
+    <Foo
+     />
+
+    // good
+    <Foo />
+    ```
+
+## Props
+  - Alphabetically order your props.
+    ```javascript
+    // bad
+    <Foo
+      className="bar baz"
+      bar="bar"
+    />
+
+    // good
+    <Foo
+      bar="bar"
+      className="bar baz"
+    />
+    ```
+
+## Parentheses
+  - Wrap JSX tags in parentheses when they span more than one line:
+    ```javascript
+    /// bad
+    render() {
+      return <MyComponent className="long body" foo="bar" >
+               <MyChild />
+             </MyComponent>;
+    }
+
+    // good
+    render() {
+      return (
+        <MyComponent className="long body" foo="bar">
+          <MyChild />
+        </MyComponent>
+      );
+    }
+
+    // good, when single line
+    render() {
+      const body = <div>hello</div>;
+      return <MyComponent>{body}</MyComponent>;
+    }
+    ```
+
+## Tags
+  - Always self-close tags that have no children.
+    ```javascript
+    // bad
+    <Foo className="stuff"></Foo>
+
+    // good
+    <Foo className="stuff" />
+    ```
+
+  - If your component has multi-line properties, close its tag on a new line.
+    ```javascript
+    // bad
+    <Foo
+      bar="bar"
+      baz="baz" />
+
+    // good
+    <Foo
+      bar="bar"
+      baz="baz"
+    />
```

Comment:
I'm not sold on the `/>` goes to new line. I know the benefit is the same as the additional trailing comma, but it adds noising lines that making scanning a `render` call harder. So the tradeoff is easier to read vs. nice git diffs.